test/driver.js

```diff
@@ -582,6 +584,18 @@ var Driver = (function DriverClosure() { // eslint-disable-line no-unused-vars
       this._log('Done !');
       this.end.textContent = 'Tests finished. Close this window!';
 
+      var re = new XMLHttpRequest();
+      re.open('POST', '/browserTestReports', false);
+      re.setRequestHeader('Content-Type', 'application/json');
+      re.responseType = 'arraybuffer';
+      re.onreadystatechange = function () {
+        if (re.readyState === XMLHttpRequest.DONE && re.status === 200) {
+          var browserTestInfo = JSON.stringify(window.__coverage__);
+          re.send(browserTestInfo);
```

Comment:
Can you rename `browserTestInfo` to a more accurate name? E.g. `coverageResults`.